nextcord/abc.py

```diff
@@ -1366,10 +1366,14 @@ async def send(
         if embed is not None and embeds is not None:
             raise InvalidArgument("Cannot pass both embed and embeds parameter to send()")
 
+        local_embed_files: List[File] = []
         if embed is not None:
+            local_embed_files = list(embed._local_files.values())
             embed = embed.to_dict()
 
         elif embeds is not None:
+            for embed in embeds:
+                local_embed_files.extend(list(embed._local_files.values()))
```

Comment:
```suggestion
                local_embed_files.extend(embed._local_files.values())
```
The `list` can be removed as `list.extend` takes an iterable